app/controllers/appointments_controller.rb

```diff
@@ -91,4 +83,13 @@ def download_filename
     facility_name = current_facility.present? ? current_facility.name.parameterize : "all"
     "overdue-patients_#{facility_name}_#{Time.current.to_s(:number)}.csv"
   end
+
+  def patient_summaries(only_overdue: true)
+    PatientSummaryQuery.call(
```

Comment:
This interface and interaction is looking a lot slicker. Does this query only service overdue lists? Can we rename the query class?